### PR TITLE
feat: add v2.1.0 upgrade handler

### PR DESCRIPTION
**Pull request by H4NLee**
Added an upgrade handler for v2.1.0(maybe) by referring to the [guide](https://docs.cosmos.network/main/core/upgrade#genesis-state).
Tested upgrade successfully and genesis state of oracle and module params.

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -707,6 +707,10 @@ func (app *App) InitChainer(ctx sdk.Context, req abci.RequestInitChain) abci.Res
 	if err := tmjson.Unmarshal(req.AppStateBytes, &genesisState); err != nil {
 		panic(err)
 	}
+
+	// set module version map
+	// https://docs.cosmos.network/main/core/upgrade#genesis-state
+	app.UpgradeKeeper.SetModuleVersionMap(ctx, app.mm.GetVersionMap())
```
Comment (H4NLee) [inline app/app.go:713]: This should have been added in the previous upgrade(v2.0.5).
It's for set module version map when initializing chain.
No effect on this upgrade.

**Review by gyuguen — APPROVED** [review]
Thank you

**Review by audtlr24 — APPROVED** [review]
LGTM, Thanks!

**Review by inchori — APPROVED** [review]
LGTM

**Pull request merged**